cfmt_test.go

```diff
@@ -21,20 +19,25 @@ type (
 	SprintlnFunc func(...interface{}) string
 )
 
+const (
```

Comment:
Hm, I don't know it is a good idea that we declare the color code ourselves in test code. I think using color code from the color library is better. But `aurora` has no ANSI color codes .. 🤔

Or I think the color codes should be calculated using the `aurora.RedFg`, `aurora.Green` and so on.

Reply:
I changed it to use `aurora.Color`. Could you check it?

Reply:
Using `aurora.Color` is good :)